app/src/services/realitio.ts

```diff
@@ -93,14 +93,18 @@ class RealitioService {
     const iface = new ethers.utils.Interface(realitioAbi)
     const event = iface.parseLog(logs[0])
 
-    const question: QuestionLog = RealitioQuestionLib.populatedJSONForTemplate(
+    const questionLog: QuestionLog = RealitioQuestionLib.populatedJSONForTemplate(
       RealitioTemplateLib.defaultTemplateForType('bool'),
       event.values.question,
     )
 
+    const category =
+      questionLog.category && questionLog.category === 'undefined' ? '' : questionLog.category
+    const question = questionLog.title && questionLog.title === 'undefined' ? '' : questionLog.title
```

Comment:
I think you can remove `questionLog.category &&` and `questionLog.title &&`.

If they are falsy, then the whole `field && field === 'undefined'` will be false, and the falsy value will be returned.

If the first operand of the `&&` is removed, and the value is falsy, then the `=== 'undefined'` will be false, and (again) the falsy value will be returned.

So, same result, less code and (IMO) more clear.

Reply:
Done